### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by thankyoumichaelbazzell** [0perationPrivacy/VoIPSuite-Mobile#53] [General]
I love that this project was brought to life and am very thankful for its existance. I have contributed a few times, but am now wondering if this project has a future or if the devs have moved onto other things. I don't want to continue using it if it will not be maintained regarding dependencies, security patches, fixing issues etc.

Question for the devs, is this something that is a lwoer priority because it is not bringing in enough donations to be sustainable? Or is it purely a time issue and you cannot dedicate it as a priority? Just looking for some insight to see how we, the community can help keep this thing alive! Thanks!

**Comment by 0perationPrivacy** [comment]
Hi, It is a resource issue. No one has volunteered to the code, I hire a developer on an hourly basis to get some of the tasks done. And it is also low priority. The OperationPrivacy Dashboard is higher priority. It has nothing to do with donations (not a lot of donations have come in. In comparison, I've gotten about $1000 in the last 3 years in donations, the cost has been over $3000 since the start paid to a developer for some of the initial core development. I do it for free in my spare time, I don't keep a dime). This is still active, hoping more people will contribute to it. I plan to get some of the other feature requests and bug fixes dealt with in the next 2-3 months. I usually let them collect and hire a dev to take care of them all at once.

**Comment by getsean** [comment]
If I donated like $200 could you please add a place to instead of choosing a phone number from the drop down list you can just enter the phone number that you want to use? All my numbers are still not showing up for me with Telnyx. It only shows me like 20 or so numbers. I have 200 numbers and the one I want to use I purchased first and it only shows me my most recent numbers for some reason.

**Reply by huzaifa3115** [reply]
The issue has been resolved with https://github.com/0perationPrivacy/VoIPSuite-Mobile/releases/tag/v-0.8

From now on, we will be proactively overseeing the project and addressing its issues.

**Comment by huzaifa3115** [comment]
Fix https://github.com/0perationPrivacy/VoIPSuite-Mobile/releases/tag/v-0.8
